ocf-servers/js-servers/environmental_sensor.js

```diff
@@ -18,17 +18,19 @@ var	device = require('iotivity-node')('server'),
 	sensorData =   {temperature : 0.0,
 			humidity : 0.0,
 			pressure : 0.0,
-			uvIndex : 0};
+			uvIndex : 0},
+    simData = 0.0;
 
 var noble = '';
 try {
 	noble = require('noble');
 }
 catch(e) {
 	debuglog('Require noble module: ', e.message);
-	process.exit(0);
+	//process.exit(0);
 }
 
+if(noble){
 // intitialize BLE, make sure BT is enabled with 'rfkill unblock bluetooth'
```

Comment:
The indentation looks a little awkward now that we've added an if loop. Can we fix this?